src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -48,6 +49,32 @@
     (normalize-input current v)
     current))
 
+(defn- routes
```

Comment:
I get the sense the routes is going to be a big feature. Can we put this into it's own view & style file as it will only get more complicated from here :)